changelogs/unreleased/3565-compatibility-iso4-modulev2.yml

```diff
@@ -0,0 +1,7 @@
+change-type: patch
+issue-nr: 3565
+description: Raise a warning when trying to use repo of type package.
+destination-branches: [iso4]
+sections:
+    bugfix: "{{description}}"
```

Comment:
I'd missed this in my earlier review. I don't think a bugfix note makes sense here. The new repo type has only just been introduced.

Reply:
It seems I was just too late. I'll fix this. @arnaudsjs do you agree the bugfix note is not appropriate?